### PR TITLE
Add forumla for fifechan.

### PR DESCRIPTION
**Pull request by stubb**

### CODE REVIEW

File: fifechan.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
+class Fifechan < Formula
+  desc "Is a C++ GUI library designed for games."
```
Comment (dunn) [inline fifechan.rb]: This can just be " C++ GUI library designed for games" (no period)

File: fifechan.rb [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  homepage "https://fifengine.github.io/fifechan/"
+  url "https://github.com/fifengine/fifechan/archive/0.1.2.tar.gz"
+  sha256 "4a4239314ae33c5413e4d7d3e5f2f1a7e53fb596fd1480ea7443ee78041e6b2d"
+
+  depends_on "cmake" => :build
+  depends_on "allegro" => :build
+  depends_on "sdl_image" => :build
```
Comment (dunn) [inline fifechan.rb]: Neither of these are needed at run-time?

**Comment by stubb** [comment]
Can someone give me a hint please why my pullrequest fails for the buildbot?
```
CONFLICT (add/add): Merge conflict in fifechan.rb
```

**Comment by dunn** [comment]
Sometimes it has trouble merging the patch.  Usually rebasing against `origin/master` fixes it.

**Comment by stubb** [comment]
@dunn Thanks that did it!

**Comment by dunn** [comment]
Merged in c255a70bad36f9d4955aa26088d0bfc14bcf9037, thank you for contributing!

**Pull request closed without merging**